flutter/lib/src/screenshot/recorder.dart

```diff
@@ -137,3 +142,63 @@ class ScreenshotRecorder {
     }
   }
 }
+
+extension on widgets.BuildContext {
+  WidgetFilterColorScheme findColorScheme() {
+    WidgetFilterColorScheme? result;
+    visitAncestorElements((el) {
+      result = getElementColorScheme(el);
+      return result == null;
+    });
+
+    if (result == null) {
+      int limit = 20;
```

Comment:
I assume 20 is an arbitrary number right?

Reply:
yes, completely